src/pages/components/select/usage.mdx

```diff
@@ -18,30 +17,75 @@ in forms for users to submit data.
 <AnchorLink>Overview</AnchorLink>
 <AnchorLink>Live demo</AnchorLink>
 <AnchorLink>Variants</AnchorLink>
-<AnchorLink>Best practices</AnchorLink>
-<AnchorLink>Accessibility</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
 ## Overview
 
-Select is a type of input that is used in forms, where a user is submitting data
-and chooses one option from a list.
+The select component collects user-provided information from a list of options.
+Selects are usually used in forms where a user submits data and chooses one
+option from a list.
 
-### Select versus Dropdown
+<Row>
+<Column colLg={8}>
+
+![Select component](images/select-usage-1.png)
+
+</Column>
+</Row>
 
-While the select and dropdown components look similar, they have different
-functions.
+#### When to use
 
 - Use the select component inside a form where users are selecting from a list
   of options and submitting data.
-- Use the dropdown component to filter or sort content on a page.
+
+- When the experience is mostly form-based.
+
+#### When not to use
+
+It is best practice not to use a select if there are fewer than three options
+for selection. In this case, use a
+[radio button](https://carbondesignsystem.com/components/radio-button/usage/)
+group instead.
+
+#### Select versus Dropdown
```

Comment:
There are some weird code things going on here in the Select versus Dropdown section.
<img width="997" alt="Screen Shot 2022-12-12 at 10 48 37 AM" src="https://user-images.githubusercontent.com/43969356/207090145-912f941a-67c7-411e-9903-e2e624b5c3d2.png">